keyboards/nk65/readme.md

```diff
@@ -0,0 +1,37 @@
+NK65
+=========
+
+[NK65]()
```

Comment:
There should be an image here of some sort, if possible. Something like:

```suggestion
![NK65](http://yiancar-designs.com/wp-content/uploads/2019/05/NK65_top.jpg)
```

Otherwise this line should be deleted IMO.

Reply:
I left it blank to be added with a nice render when I have one :) I can comment it out right now or leave it as an empty link and add the pic when I got it

Reply:
Just as long as it doesn't get forgotten. Commenting out might work too, but I don't know what GitHub does if you try to comment out an image.

Reply:
Lets go as an empty link then if that's OK

Reply:
:thinking: ...

Yeah, that's fine.